utils/node_local_test.py

```diff
@@ -4486,6 +4486,83 @@ def test_mount_ro(self):
                 return
             raise
 
+    def import_torch(self):
+        """Return a handle to the pydaos.torch module"""
+
+        os.environ['DD_MASK'] = 'all'
+        os.environ['DD_SUBSYS'] = 'all'
+        os.environ['D_LOG_MASK'] = 'DEBUG'
+        os.environ['FI_UNIVERSE_SIZE'] = '128'
+        os.environ['DAOS_AGENT_DRPC_DIR'] = self.conf.agent_dir
+
+        import importlib
+        return importlib.import_module('pydaos.torch')
```

Comment:
i am not sure the CI VM has the relevant pytorch module installed. We probably need to fix that and also update the docker file to install what we need.

Reply:
I've added into `utils/cq/requirements.txt` which resolved errors for `daos_pylint.py`.
There are two more `requirements-ftest.txt` and `requirements-utest.txt` I think I'll need to add torch package to them as well ?